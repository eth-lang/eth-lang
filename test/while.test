; vim: set ft=clojure:

(while true (do-something))

(while (< i 5) (show 5) (set i (+ i 1)))
