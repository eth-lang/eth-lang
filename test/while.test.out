(function() { var __eth__result; while (true) {
  __eth__result = doSomething();
} return __eth__result; })();
(function() { var __eth__result; while ((i < 5)) {
  show(5);
  __eth__result = i = (i + 1);
} return __eth__result; })();

